woocommerce/woocommerce
# Post external smoke test results to GitHub PR

### All Submissions:






### Changes proposed in this Pull Request:

This PR takes the results from running smoke tests against an external site and posts it as a comment on the associated PR. This action get triggered when the `run: smoke tests` label is added to the PR.

Closes #30615 .

### How to test the changes in this Pull Request:

1. Create a PR
2. Add `run: smoke tests` label
3. Wait for tests to complete and observe comments on the associated PR
